service/src/main/resources/messages_en.yaml

```
@@ -0,0 +1,134 @@
+  Please ask your question — and I’ll call one of my human colleagues ✍️👇
+  If you want to return to my functions after talking to a human,
+  just click "End conversation with manager"
+
+green.office.subject: "Interested in Green Office"
+unknown.error: "An unknown error occurred, please try again"
+message.sent.to.manager: "Your message has been sent to the manager, please wait for a response"
+client.want.to.speak: "👨@%s wants to speak with a human\n\n👉 %s\n\n💬 http://localhost:4200/chat/%s"
+previous.session.expired: "Previous session expired. Please choose a command from the list to continue"
+photo.content: "Photo content"
+login.failed: "Incorrect login or password. Please check your credentials and try again."
+client.support.message.change.language: "Language was switched"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)